src/liquid.ts

```diff
@@ -74,7 +74,7 @@ export class Result {
   constructor(
     readonly name: string,
     readonly percentage: number,
-    readonly ml: number,
+    readonly amount: number,
```

Comment:
This change will break a lot of code, whole Liquid Calculator expect `.ml`, not `.amount`. Do you want to fix that code ? I have better suggestion, let's create another type `MixableResult` that will be used only in mixer. Then you can modify whatever you want ok ?

Reply:
sure
@stasbar